Makefile

```diff
@@ -21,11 +21,11 @@ OBJS += acados/dense_qp/dense_qp_qore.o
 OBJS += acados/ocp_qp/ocp_qp_common.o
 OBJS += acados/ocp_qp/ocp_qp_common_frontend.o
 OBJS += acados/ocp_qp/ocp_qp_hpipm.o
-OBJS += acados/ocp_qp/ocp_qp_condensing.o
 OBJS += acados/ocp_qp/ocp_qp_partial_condensing.o
+OBJS += acados/ocp_qp/ocp_qp_full_condensing.o
 OBJS += acados/ocp_qp/ocp_qp_sparse_solver.o
```

Comment:
we should probably rename the sparse_solver to partial_condensing_solver too, to be consistent.